packages/smooth_app/lib/widgets/smooth_product_carousel.dart

```diff
@@ -326,3 +328,88 @@ class _SearchCardTagLineDefaultText extends StatelessWidget {
     );
   }
 }
+
+class _SearchCardTagLineDeprecatedAppText extends StatelessWidget {
+  const _SearchCardTagLineDeprecatedAppText({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations localizations = AppLocalizations.of(context);
+
+    return Padding(
+      padding: const EdgeInsets.symmetric(
+        horizontal: 10.0,
+      ),
+      child: TextButton(
+          onPressed: () {
+            _openAppStore();
+          },
+          child: Text(
+            '${localizations.depricatedHeader}  ${localizations.clickHereToDownload}',
```

Comment:
It would make more sense to have the header as a `Text` and then the "click!" as a `TextButton`.

Reply:
I did try it first like this but would involve more lines of code but I agree would be better for the user. In the request of the issue, the line on the UI requested was all in one line. So I added it to the whole text as a TextButton.